obr_core/donation/api/views.py

```
@@ -0,0 +1,577 @@
+
+
+class SingleDonationReturnView(
+    APIView,
+):
+
+    @extend_schema(
+        operation_id="donation_single_return",
+        responses={status.HTTP_302_FOUND: None},
+    )
+    def get(self, request):
+        next = request.GET.get("next", "/")
```

Ruff (0.11.9)

Variable `next` is shadowing a Python builtin

(A001)